pytab/core.py

```diff
@@ -300,6 +307,32 @@ def cli(
     system_info = hwi.get_system_info()
     click.echo(" success!")
 
+    # Logic for GPU Selection
+    gpus = system_info["gpu"]["gpu_elements"]
+    if len(gpus) > 1 and "gpu_id" not in globals():
+        click.echo("| Multiple GPU's detected")
+        for gpu in gpus:
+            click.echo(f"| {gpu["id"]}: {gpu["name"]} ")
+        gpu_id = click.prompt(
+            f"| please select a gpu id\n| Must be less than {len(gpus)}", type=int
+        )
+        if gpu_id > len(gpus):
```

Comment:
Note: This Check only tests wether the Value is too high or not. Therefore values below 0 can be entered.

Instead it should be set an exact in bounce range. 
E.g. `if not (0 <= gpu_id < len(gpus))`